helper.php

```diff
@@ -392,6 +397,23 @@ function _descCell() {
     }
 
     /**
+     * Diff icon / link to diff page
+     */
+    function _diffCell($id) {
+        // check for page existence
+        if (!isset($this->page['exists'])) {
+            if (!isset($this->page['file'])) $this->page['file'] = wikiFN($id);
+            $this->page['exists'] = @file_exists($this->page['file']);
+        }
+
+        // produce output
+        $content = '<a href="'.wl($id).($this->page['section'] ? '#'.$this->page['section'] : '').'&do=diff" class="diff_link">
+<img src="/lib/images/diff.png" width="15" height="11" title="Show differences to current revisions" alt="Show differences to current revisions"/>
```

Comment:
Please add these strings to the language files such that they can be translated.